src/FluentMigrator.DotNet.Cli/Commands/Migrate.cs

```diff
@@ -23,13 +23,9 @@ namespace FluentMigrator.DotNet.Cli.Commands
 {
     [HelpOption]
     [Command("migrate", Description = "Apply migrations")]
-    [Subcommand(typeof(MigrateUp))]
-    [Subcommand(typeof(MigrateDown))]
+    [Subcommand(typeof(MigrateUp), typeof(MigrateDown))]
```

Comment:
This change is nice too. Keep it.

Reply:
I don't 100% agree.  @gandarez  Please see https://github.com/fluentmigrator/fluentmigrator/issues/1033 for how I envisioned supporting sub-commands for up and down.  The annoying part is MigrationCommand has --start-version.  I'd like to alias --start-version to --minimum-version.  @fubar-coder  Thoughts?

Reply:
Why this change is related to that issue @jzabroski ?

Reply:
Yes, we should discuss the matter of `--minimum-version` vs `--start-version` in a different issue.

BTW: Just a hint for future contributions: Don't introduce style changes that don't directly contribute to the problem the PR tries to solve. This will only complicate the review process and makes it less likely that the PR will get merged.

Reply:
Yes -- I started looking into whether Mcmaster.commandlineutils supports aliases.  Even if it doesnt, probably an easy PR to merge into his project. But a bit more yak shaving.